azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -1403,6 +1446,20 @@ func expandKubernetesClusterRoleBasedAccessControl(input []interface{}, provider
 	return rbacEnabled, aad
 }
 
+func expandKubernetesClusterManagedClusterIdentity(input []interface{}) *containerservice.ManagedClusterIdentity {
+	if len(input) == 0 || input[0] == nil {
+		return nil
+	}
+
+	val := input[0].(map[string]interface{})
```

Comment:
would `values` make more sense?
```suggestion
	values := input[0].(map[string]interface{})
```